neurotech/index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="/assets/external/bootstrap.min.css">
+        <link rel="stylesheet" href="/assets/external/bootstrap-icons.css">
+        <link rel="stylesheet" href="/assets/ieee.css">
+        <link rel="stylesheet" href="./neurotech.css">
+        <link rel="icon" href="/favicon.ico"/>
+
+        <meta name="description" content="The official website of the Purdue IEEE Student Branch">
+        <meta name="author" content="Purdue IEEE">
+
+        <title>Purdue IEEE</title>
+
+        <script src="/assets/external/bootstrap.bundle.min.js"></script>
+        <script src="/assets/external/vue2.min.js"></script>
+        <script src="/assets/include.js"></script>
+
+    </head>
+    <body>
+    <div id="app" class="bg-dark">
+        <ieee-header head="Purdue IEEE" page="Neurotech"></ieee-header>
+
+        <main>
+            <div class="container-fluid pt-5 mt-5 text-center bg-dark">
+                <img src="./images/ntc-logo-white.png" alt="Neurotech Logo" width="300">
+                <p class="display-2 text-light">Neurotech Committee</p>
+                <p class="fs-3 text-light">The IEEE Neurotech Committee (NTC) works on creating Brain Machine Interfaces using Machine Learning and Electroencephalogram technology. Designing signal processing networks, programming neural networks and conducting human testing are all core aspects of the NTC.</p>
+                <div class="container py-5">
+                    <div class="row justify-content-center align-items-center g-2 mb-5">
+                        <div class="col-4">
+                            <img src="./images/carousel/1.jpg" alt="Neurotech Photo 1" width="400">
+                        </div>
+                        <div class="col-4">
+                            <img src="./images/carousel/2.jpg" alt="Neurotech Photo 2" width="400">
+                        </div>
+                        <div class="col-4">
+                            <img src="./images/carousel/3.jpg" alt="Neurotech Photo 3" width="400">
+                        </div>
+                    </div>
+                    <div class="row justify-content-center align-items-center g-2">
+                        <div class="col-4">
+                            <img src="./images/carousel/4.jpg" alt="Neurotech Photo 4" width="400">
+                        </div>
+                    </div>
+                </div>
+                <p class="display-6 text-light">Reach out to <a href="mailto:asultanb@purdue.edu">asultanb@purdue.edu</a> if interested.</p>
+                <p class="fs-4 text-light">Return to the <a href="/">home page</a>.</p>
```

Comment:
This link to the home page is completely unnecessary and not standard to the other pages.

Reply:
resolved